openedx/adg/lms/applications/tests/test_models.py

```diff
@@ -469,7 +579,36 @@ def test_multilingual_course_group_is_business_line_prerequisite_property(busine
     """
     multilingual_course_group = MultilingualCourseGroupFactory(is_program_prerequisite=False)
 
-    if business_line:
-        multilingual_course_group.business_line_prerequisite = BusinessLineFactory()
+    if not business_line:
+        multilingual_course_group.business_line_prerequisite = None
 
     assert multilingual_course_group.is_business_line_prerequisite == expected_result
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize('number_of_users', [2, 4, 6])
+def test_get_enrollments_and_completions_count(course_group_with_courses, number_of_users):
+    """
+    Test that the total number of enrollments and completions in all the
+    courses of a course group are returned.
+    """
+    users = UserFactory.create_batch(number_of_users)
+    courses = course_group_with_courses.multilingual_courses.open_multilingual_courses()
+
+    for idx, user in enumerate(users):
+
+        course_index = 0
+        if idx % 2 == 1:
+            course_index = 1
+
+        CourseEnrollmentFactory(user=user, course=courses[course_index].course)
+        PersistentCourseGrade.update_or_create(
+            user_id=user.id,
+            course_id=courses[course_index].course.id,
+            percent_grade=80,
+            letter_grade='A',
+            passed=True,
+        )
```

Comment:
We have a factory for this in performance/tests/factories.py. Maybe we can use that here?